content/articles/android/index.md

```diff
@@ -51,15 +51,14 @@ However this may take quite a while - each docker invocation will do a clean bui
 
 One way to make it a little bit faster: add ` -v /tmp/registry\":/usr/local/cargo/registry\"` to a docker command. This will tell docker to use `/tmp/registry` on the host machine for cargo's registry, therefore docker will not download all the dependencies on each build.
 
-Another possibility: run docker interactively and invoke build command in the same container for each build.
+If you get an error similar to: `failed to select a version for the requirement hashbrown = "=0.12.3"`
 ```
 docker run 
   --rm 
   -v $(pwd):/root/src 
   -w /root/src 
```

Comment:
Nice tip with `cargo update`, but the section was about running build multiple time in the same container

with the change it do just the same thing as the command from the previous section?

Reply:
Oh i see. Maybe we could just add it to the article if that doesnt make it too long